src/main/flight/pid.c

```diff
@@ -892,13 +896,13 @@ static void applyItermRelax(const int axis, const float ITerm,
 
     if (itermRelax &&
        (axis < FD_YAW || itermRelax == ITERM_RELAX_RPY ||
-       itermRelax == ITERM_RELAX_RPY_INC)) {
+        itermRelax == ITERM_RELAX_RPY_INC)) {
         itermRelaxIsEnabled = true;
         const float itermRelaxFactor = 1 - setpointHpf / ITERM_RELAX_SETPOINT_THRESHOLD;
 
         const bool isDecreasingI =
             ((ITerm > 0) && (*itermErrorRate < 0)) || ((ITerm < 0) && (*itermErrorRate > 0));
-        if ((itermRelax >= ITERM_RELAX_RP_INC) && isDecreasingI) {
+        if ((itermRelaxType >= ITERM_RELAX_RP_INC) && isDecreasingI) {
```

Comment:
I think this is incorrect and there wasn't any bug to fix.

`itermRelax` is type:
```
typedef enum {
    ITERM_RELAX_OFF,
    ITERM_RELAX_RP,
    ITERM_RELAX_RPY,
    ITERM_RELAX_RP_INC,
    ITERM_RELAX_RPY_INC
} itermRelax_e;
```
and determines which axes and how it applies (under increasing only or all conditions).

`itermRelaxType` is type:
```
typedef enum {
    ITERM_RELAX_GYRO,
    ITERM_RELAX_SETPOINT
} itermRelaxType_e;
```
and determines the method in which term relax operates.